### ISSUE TITLE
Can't equip Pointshop2 items

### ISSUE THREAD

**Issue by BenitoTV** [TTT-2/TTT2#1270]
## Your version of TTT2 (mandatory)
Workshop version from steam: v0.12.2b

## Describe the bug (mandatory)

When a player tries to equip any item from the Pointshop (Hat, Trail, Playermodel, anything), a pop-up appears that says that the item couldn't be equipped because of a lua error. This problem is present since the latest big update (I think it was v0.12.1b). I already opened an issue on the github of Pointshop2, but the creator said the following: 

> "Looks like some code (TTT2 perhaps since you mentioned the update?) is using callback as a global variable which breaks any scripts trying to use that name. Not really anything I can do here - it needs to be fixed in the other addon"

So I hope I find a solution here.

## To reproduce

Steps to reproduce the behaviour:

1. Open the Pointshop2 menu with the F3 key
2. Buy an item in the "Shop" tab
3. Equip the item in the "Inventory" tab
5. See the error in the console

## Expected behaviour

The item should be equipped and visible to other players, without an error pop-up.

## Context (please provide as much as you can)

- Collection link of your Server: https://steamcommunity.com/sharedfiles/filedetails/?id=2791902327
- Console error: 
```
L 12/25/2023 - 21:13:06: Lua Error:
        addons/libk/lua/libk/server/sv_libk_database.lua:171: attempt to call local 'callback' (a table value)
                 1: Query (sqlText = "UPDATE `kinv_items` SET  `itemclass` = 'KInventory.Items.91',`data` = '{\"purchaseData\":{\"amount\":0.0,\"time\":1702416149.0,\"currency\":\"points\",\"origin\":\"LUA\"}}',`itempersistence_id` = '91',`inventory_id` = NULL WHERE `id` = '119'", callback = _R._LOADED.team:callback, errorCallback = function (error), blocking = nil) [addons/libk/lua/libk/server/sv_libk_database.lua: 171]
                 2: add (self = { _R._LOADED.team:LibK.TransactionSqlite.__instanceDict: 0xe9e84c72 }, sqlText = "UPDATE `kinv_items` SET  `itemclass` = 'KInventory.Items.91',`data` = '{\"purchaseData\":{\"amount\":0.0,\"time\":1702416149.0,\"currency\":\"points\",\"origin\":\"LUA\"}}',`itempersistence_id` = '91',`inventory_id` = NULL WHERE `id` = '119'") [addons/libk/lua/libk/server/sv_database_transaction.lua: 77]
                 3: s (slot = { _R._LOADED.team:Pointshop2.EquipmentSlot.__ins
        addons/libk/lua/libk/server/sv_libk_database.lua:171: attempt to call local 'callback' (a table value)
                 1: Query (sqlText = "UPDATE `kinv_items` SET  `itemclass` = 'KInventory.Items.91',`data` = '{\"purchaseData\":{\"amount\":0.0,\"time\":1702416149.0,\"currency\":\"points\",\"origin\":\"LUA\"}}',`itempersistence_id` = '91',`inventory_id` = NULL WHERE `id` = '119'", callback = _R._LOADED.team:callback, errorCallback = function (error), blocking = nil) [addons/libk/lua/libk/server/sv_libk_database.lua: 171]
                 2: add (self = { _R._LOADED.team:LibK.TransactionSqlite.__instanceDict: 0xe9e84c72 }, sqlText = "UPDATE `kinv_items` SET  `itemclass` = 'KInventory.Items.91',`data` = '{\"purchaseData\":{\"amount\":0.0,\"time\":1702416149.0,\"currency\":\"points\",\"origin\":\"LUA\"}}',`itempersistence_id` = '91',`inventory_id` = NULL WHERE `id` = '119'") [addons/libk/lua/libk/server/sv_database_transaction.lua: 77]
                 3: s (slot = { _R._LOADED.team:Pointshop2.EquipmentSlot.__instanceDict: 0xebb9feea }) [addons/poiLUA Error in Controller Pointshop2Controller action equipItem:

nil
```

- Other addons installed in your servers `addons` folder: Pointshop2, PAC3, LibK
- Screenshot: 
![image](https://github.com/TTT-2/TTT2/assets/111009199/d966a1f6-a86c-4699-89a5-560b66056786)

**Comment by PunishMints** [comment]
I'm having the same issue with a brand new server I started up almost a week ago. I also have Pointshop2, PAC3, and LibK installed in the addons folder along with some Pointshop 2 DLC (Acheivements, Betting, Trading, Boosters, Gambling, Drops, and PermaWeapons)

Here is the [link to my collection.](https://steamcommunity.com/sharedfiles/filedetails/?id=3119625446)

**Comment by Histalek** [comment]
Thank you for reporting this!

This was fixed in #1261 but is not released to the workshop yet. We plan on making a release in the next few days.

For now the workaround would be to download the master branch from github and place it into the `/garrysmod/addons/` directory. If you are on Linux make sure the directory is in lowercase e.g. `ttt2` as otherwise garrysmod won't recognize it.

I'll close this Issue as completed, but will pin it for the time being.

**Comment by Histalek** [comment]
Update v0.12.3b is now released on the workshop and contains the fix for this issue :)
